physics/smoke_dust/rrfs_smoke_wrapper.F90

```diff
@@ -123,7 +123,7 @@ subroutine rrfs_smoke_wrapper_run(im, kte, kme, ktau, dt, garea, land, jdate,
                    ebu_smoke,fhist,min_fplume,                                             &
                    max_fplume, hwp, hwp_ave, wetness, ndvel, ddvel_inout,                  &
                    peak_hr_out,lu_nofire_out,lu_qfire_out,                                 &
-                   fire_heat_flux_out, frac_grid_burned_out, kpbl,oro,                     &
+                   fire_heat_flux_out, frac_grid_burned_out, kpbl,oro, totprcp,            &
```

Comment:
Why are you adding `totprcp` when you're not using it?

Reply:
@grantfirl I will remove this unused variable. Thanks.